src/highs.jl

```diff
@@ -5,15 +5,16 @@ using HiGHS: Highs_create, Highs_setStringOptionValue, Highs_setSolution,
              Highs_getModelStatus, HighsInt
 
 function solve_IP(::Val{:highs}, O::OptProblem, initial_solution = Int[],
-                  use_warmstart = true; options...)
+                  use_warmstart = true; solver_options, options...)
```

Comment:
The `options` argument looks unused now.

Reply:
The only instance now is in the`cbc.jl` code, but I can remove that too.

Reply:
Right, I can line that up.